synfig-studio/src/gui/dialogs/canvasproperties.cpp

```diff
@@ -165,12 +165,12 @@ CanvasProperties::CanvasProperties(Gtk::Window& parent,etl::handle<synfigapp::Ca
 	//Gtk::Table *table=manage(new Gtk::Table());
 	//table->attach(*scrolledwindow, 0, 2, 0, 1, Gtk::EXPAND|Gtk::FILL, Gtk::EXPAND|Gtk::FILL, 0, 0);
 
-	//Gtk::Button* button_add(manage(new Gtk::Button(Gtk::StockID("gtk-add"))));
+	//Gtk::Button* button_add(manage(new Gtk::Button("gtk-add")));
```

Comment:
It isn't the correct text ("Add"?) and isn't translatable this way (Although it's commented out :P)